### Data Warehouse and data mining systems #2

In [4]:
import pandas as pd
import numpy as np 

DATA_DIR="SpotifyAnalysis\\datasets\\"

In [9]:
data1 = pd.read_csv("".join([DATA_DIR,"SpotifyTop200songs2017-2021\\data.csv"]), sep="#")
data2 = pd.read_excel("D:\Repos\SpotifyAnalysis\SpotifyAnalysis\datasets\Spotify 2010 - 2019 Top 100 Songs.xlsx")
print(data1.head(5))
print('-'*120)
print(data2.head(5))



   Position                               Track Name            Artist  \
0         1                                  Starboy        The Weeknd   
1         2                                   Closer  The Chainsmokers   
2         3                          Let Me Love You          DJ Snake   
3         4  Rockabye (feat. Sean Paul & Anne-Marie)      Clean Bandit   
4         5                                One Dance             Drake   

   Streams        Date                                              Genre  
0  3135625  2017-01-01  ['canadian pop', 'canadian contemporary r&b', ...  
1  3015525  2017-01-01  ['pop', 'pop dance', 'tropical house', 'edm', ...  
2  2545384  2017-01-01  ['pop', 'electronic trap', 'dance pop', 'edm',...  
3  2356604  2017-01-01  ['pop', 'uk dance', 'dance pop', 'uk funky', '...  
4  2259887  2017-01-01  ['toronto rap', 'canadian pop', 'canadian hip ...  
---------------------------------------------------------------------------------------------------


#









<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Position</th>
      <th>Track Name</th>
      <th>Artist</th>
      <th>Streams</th>
      <th>Date</th>
      <th>Genre</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>1</td>
      <td>Starboy</td>
      <td>The Weeknd</td>
      <td>3135625</td>
      <td>2017-01-01</td>
      <td>['canadian pop', 'canadian contemporary r&amp;b', 'pop']</td>
    </tr>
    <tr>
      <th>1</th>
      <td>2</td>
      <td>Closer</td>
      <td>The Chainsmokers</td>
      <td>3015525</td>
      <td>2017-01-01</td>
      <td>['pop', 'pop dance', 'tropical house', 'edm', 'electropop', 'dance pop']</td>
    </tr>
    <tr>
      <th>2</th>
      <td>3</td>
      <td>Let Me Love You</td>
      <td>DJ Snake</td>
      <td>2545384</td>
      <td>2017-01-01</td>
      <td>['pop', 'electronic trap', 'dance pop', 'edm', 'pop dance', 'pop rap']</td>
    </tr>
    <tr>
      <th>3</th>
      <td>4</td>
      <td>Rockabye (feat. Sean Paul &amp; Anne-Marie)</td>
      <td>Clean Bandit</td>
      <td>2356604</td>
      <td>2017-01-01</td>
      <td>['pop', 'uk dance', 'dance pop', 'uk funky', 'tropical house', 'pop dance', 'post-teen pop', 'edm']</td>
    </tr>
    <tr>
      <th>4</th>
      <td>5</td>
      <td>One Dance</td>
      <td>Drake</td>
      <td>2259887</td>
      <td>2017-01-01</td>
      <td>['toronto rap', 'canadian pop', 'canadian hip hop', 'rap', 'pop rap', 'hip hop']</td>
    </tr>
  </tbody>
</table>